src/app/shared/components/category-navigation/category-navigation.component.html

```diff
@@ -1,16 +1,28 @@
 <nav class="navigation">
   <div class="categories">
     <ng-container *ngFor="let category of categories">
-      <app-type-of-help
-        [icon]="category.icon"
-        [name]="category.name"
-        [path]="['', outputPath, routingCategoryName[category.name]]"
-        [selected]="routingCategoryName[category.name] === activeRoute()"
-        [disabled]="category.disabled"
-        [redirectOnClick]="true"
-        [borderOnHover]="true"
+      <a
+        *ngIf="!category.disabled"
+        class="category-link"
+        [routerLink]="'/' + ['', outputPath, routingCategoryName[category.name]].join('/')"
```

Comment:
hmmm mayby
`[routerLink]="['/', outputPath, routingCategoryName[category.name]]"`

feel free to resolve if you tried this.

Reply:
Thanks, that's much clearer.

Reply:
Done.